components/blocks/ResendConfirmationBar/ResendConfirmationBar.vue

```diff
@@ -28,8 +27,40 @@ async function resend() {
 </script>
 
 <template>
-  <div>
-    <button @click="resend">Confirm</button>
-    <p v-if="showErrorText" class="text-red-600">{{ errorText }}</p>
+  <div class="resend-confirmation-bar__container">
+    <div class="resend-confirmation-bar__content">
+      <div class="resend-confirmation-bar__text">
+        <i-svg-circle-info />
+        <span>{{ $t('needToConfirm') }}</span>
+      </div>
+      <BaseButton class="resend-confirmation__button" @click="resend">
+        {{ $t('resendConfirmation') }}
+      </BaseButton>
+    </div>
+    <p v-if="showErrorText" class="error-text">{{ errorText }}</p>
   </div>
 </template>
+
+<style lang="postcss" scoped>
+.resend-confirmation-bar__container,
+.resend-confirmation-bar__content {
+  @apply flex flex-col items-center justify-center;
+}
+
+.resend-confirmation-bar__content {
+  @apply mt-4 gap-y-2 md:flex-row md:gap-x-2;
+  @apply fill-blue-500;
+}
+
+.resend-confirmation-bar__text {
+  @apply flex items-center gap-x-2;
+}
+
+.resend-confirmation__button {
+  @apply mx-1 py-1 bg-blue-100 hover:bg-blue-300;
```

Comment:
Think it's overkill to set an `active:` background colour or nah?

Reply:
I don't think so, I can add that in

Reply:
Actually don't bother tbh. There's really no need to.